spec/imsc-hrm.html

```diff
@@ -686,14 +681,17 @@ <h2>Paint Images</h2>
 
       <p>NSIZ(I<sub>i</sub>) is the number of pixels of <a>presented image</a> I<sub>i</sub>.</p>
 
-      <p>The contents of the Decoded Image Buffer D<sub>n</sub> are transferred instantaneously to Decoded Image Buffer
-      D<sub>n-1</sub> at the presentation time of <a>Intermediate Synchronic Document</a> E<sub>n</sub>.</p>
+      <p>At the presentation time of <a>Intermediate Synchronic Document</a> E<sub>n</sub>:</p>
+      <ol>
+        <li>purge from the Decoded Image Cache all images not flagged <i>retain</i>; and</li>
+        <li>remove the <i>retain</i> flag from all remaining images; and</li>
+      </ol>
```

Comment:
It would be worth clarifying that this is a sequence of steps whose order is important.